src/Generator.cpp

```diff
@@ -251,22 +251,44 @@ int generate_filter_main(int argc, char **argv, std::ostream &cerr) {
     }
 
     const auto target_string = generator_args["target"];
-    const Target target = parse_target_string(target_string);
+    auto target_strings = split_string(target_string, ",");
+    std::vector<Target> targets;
+    for (const auto &s : target_strings) {
+        targets.push_back(parse_target_string(s));
+    }
 
     if (!runtime_name.empty()) {
+        if (targets.size() != 1) {
+            cerr << "Only one target allowed here";
```

Comment:
Everything else in this function uses cerr (it's essentially a main() function), so this is actually correct.